manifests/plugin/write_prometheus.pp

```diff
@@ -1,5 +1,14 @@
+# Class: collectd::plugin::write_prometheus
+#
+# @see https://collectd.org/wiki/index.php/Plugin:Write_Prometheus
+
+# Configures write_prometheus plugin.
+#
+# @param port Stdlib::Port Defines the port on which to accept scrape requests from Prometheus. Default: 9103
+# @param ip Optional[Stdlib::IP::Address] Defines the IP address to bind to. In not specified, the listener will bind to all IPs present.
```

Comment:
```suggestion
# @param port Defines the port on which to accept scrape requests from Prometheus. Default: 9103
# @param ip Defines the IP address to bind to. In not specified, the listener will bind to all IPs present.
```

Reply:
I was using the SNMP plugin documentation as the example. I'm ok with this change.